controllers/tools/index.js

```diff
@@ -103,4 +109,30 @@ router.route('/order-stats').get(injectMerchandise({ locale: 'en', showAll: true
     }
 });
 
+router.route('/applications/:formId/:applicationId?').get(async (req, res, next) => {
```

Comment:
I know not everything is broken in this controller, but maybe a candidate to live in a separate file?